tools/explorer/lib/model.py

```diff
@@ -1,12 +1,20 @@
-"""Helper Field model classes for Ontology explorer."""
+"""Helper Field model classes for Ontology explorer"""
+from typing import List
+import re
 
+from yamlformat.validator.entity_type_lib import EntityType
+from yamlformat.validator.field_lib import FIELD_CHARACTER_REGEX
+
+FIELD_INCREMENT_REGEX = re.compile(r'((?:_[0-9]+)+)$')
 
 class StandardField(object):
-  """A class to represent a generic field without increment or optionality.
+  """
+    A class to represent a generic field without increment or optionality.
 
     Args:
-        namespace_name: the name of the field's namespace as a string
-        standard_field_name: the name of the field as a string
+        namespace_name: a field's defined namespace as a string
+        standard_field_name: the un-incremented name of the field as a string.
```

Comment:
you can take any standard field name and remove the increment in the class?

Reply:
@charbull what do you mean by this?

Reply:
your class can accept a standard field name with an increment and clean it up by removing the increment. if you want to be more flexible. but it can also be left as it is.